README.md

```diff
@@ -34,9 +34,20 @@ You can configure the default behavior through various settings in your `setting
 }
 ```
 
+You can also set a new keyboard shortcut for the command in `keybindings.json`:
+
+```json
+[
+  {
+    "key": "cmd+n",
+    "command": "extension.createNewFile"
```

Comment:
aside: are all extensions forced to use the `extension` namespace for commands and not provider their own? Or was that just the example plugin that remained unchanged?